service/src/main/java/com/myhome/domain/SecurityToken.java

```diff
@@ -0,0 +1,27 @@
+package com.myhome.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import java.util.Date;
+
+@Entity
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class SecurityToken extends BaseEntity {
+  @Column(nullable = false)
+  @Enumerated(EnumType.STRING)
+  private MyHomeTokenType tokenType;
+  @Column(nullable = false, unique = true)
+  private String token;
+  @Column(nullable = false)
+  private Date creationDate;
+  @Column(nullable = false)
+  private Date expiryDate;
```

Comment:
Would be also good to add `isUsed` property to allow user use token one time only even if the token is not expired